jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlConfigurationTest.java

```diff
@@ -1622,15 +1622,23 @@ public void testCallResolvePath(String configValue, String expectedPath) throws
                     "  </Set>" +
                     "</Configure>");
 
-        configuration.setJettyStandardIdsAndProperties(null, Resource.newResource(war));
-        configuration.getProperties().put("jetty.base", "/var/lib/jetty-base");
-        if (configValue != null)
-            configuration.getProperties().put("jetty.sslContext.keyStorePath", configValue);
+        try
+        {
+            configuration.setJettyStandardIdsAndProperties(null, Resource.newResource(war));
+            configuration.getProperties().put("jetty.base", "/var/lib/jetty-base");
+            if (configValue != null)
+                configuration.getProperties().put("jetty.sslContext.keyStorePath", configValue);
 
-        TestConfiguration tc = new TestConfiguration();
-        configuration.configure(tc);
+            TestConfiguration tc = new TestConfiguration();
+            configuration.configure(tc);
 
-        assertThat(tc.getTestString(), is(expectedPath));
+            assertThat(tc.getTestString(), is(expectedPath));
```

Comment:
Can you add some windows only test cases that passing something like `C:opt\jetty` and `C:\\\opt\jetty` (probably bad examples, but you get the idea).